db/schema.rb

```diff
@@ -593,7 +593,7 @@
   add_foreign_key "broadcasts", "missions", name: "broadcasts_mission_id_fkey", on_update: :restrict, on_delete: :restrict
   add_foreign_key "choices", "answers", name: "choices_answer_id_fkey", on_update: :restrict, on_delete: :restrict
   add_foreign_key "choices", "options", name: "choices_option_id_fkey", on_update: :restrict, on_delete: :restrict
-  add_foreign_key "conditions", "form_items", column: "ref_qing_id", name: "conditions_ref_qing_id_fkey", on_update: :restrict, on_delete: :restrict
+  add_foreign_key "conditions", "form_items", column: "left_qing_id"
```

Comment:
Why were the extra params removed here?

Reply:
Good question! I'll look into it.

Reply:
Looks like Rails changed the way these are generated at some point. Doesn't make a difference practically.